ocamltest/ocamltest_stdlib.ml

```diff
@@ -177,8 +177,8 @@ module Sys = struct
 end
 
 module StringSet = struct
-  include Set.Make (String)
+  include Misc.StringSet
   let string_of_stringset s = String.concat ", " (elements s)
```

Comment:
(Following comment above.)  This should probably be [`Misc.Stdlib.`]`String.Set.to_string`.